lib/compress.js

```diff
@@ -2582,6 +2598,18 @@ merge(Compressor.prototype, {
 
     OPT(AST_Call, function(self, compressor){
         var exp = self.expression;
+        if (compressor.option("reduce_vars")
+            && exp instanceof AST_SymbolRef) {
+            var def = exp.definition();
+            if (def.fixed) {
+                exp = def.fixed;
+                if (compressor.option("unused")
+                    && def.references.length == 1
+                    && exp instanceof AST_Function) {
```

Comment:
> It would be a shallow clone, I assume. Might that present problems with multiple references to it? Do you only clone functions returning constants?

Haven't thought about that much, but since `AST_Node` doesn't reference their parents (we need to call `TreeWalker.parent()` for that), I assumed multiple reference is okay.

Come to think of it, `lib/output.js` might get confused when processing comments.